docker-tests/entry.sh

```diff
@@ -30,35 +59,56 @@ function mount_dev()
 	mount -t debugfs nodev /sys/kernel/debug
 }
 
-function start_udev()
+function init_systemd()
```

Comment:
where did you get this from ?
Impossible to validate to me ...

Reply:
This is the original file from balena's repo that I found.